helm/cas-registration/templates/reporting/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if (.Values.reportingFrontend.enabled) }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cas-registration.fullname" . }}-reporting-frontend
```

Comment:
This is going to read as `cas-registration-reporting-frontend`.Perhaps we should change the chart name to something more generic?

Reply:
ah I see there is an override in the values.yaml for this :)